CMakeLists.txt

```diff
@@ -157,9 +162,17 @@ add_subdirectory(src)
 if (ENABLE_TESTS)
   add_subdirectory(tests)
 endif ()
-if (ENABLE_BENCHMARKS)
+
+if (ENABLE_DEVELOPER_BENCHMARKS)
   add_subdirectory(benchmarks)
+  if ((NOT CMAKE_BUILD_TYPE) OR (NOT ${CMAKE_BUILD_TYPE} STREQUAL "Release"))
+    message("-- Warning: Please set CMAKE_BUILD_TYPE=Release when benchmarking!")
+  endif()
+else()
+  message("-- Warning: Benchmarks will not be built. If you want to build with benchmarks,\n"
```

Comment:
I don't think this warning is necessary.

Reply:
I thought we mentioned this specifically yesterday. Like that "enable_benchmarks" doesn't actually enable the benchmarks... maybe better for documentation stuff?